ci/run_ci.sh

```diff
@@ -61,7 +61,7 @@ module use $GDAS_MODULE_USE
 module load GDAS/$TARGET
 echo "---------------------------------------------------" >> $outfile
 rm -rf log.ctest
-ctest -R gdasapp --output-on-failure &>> log.ctest
+ctest -j12 -R gdasapp --output-on-failure &>> log.ctest
```

Comment:
Can we make the number of tasks platform dependent?

`ctest -j$NTASKS_TESTS`...
In case we need fewer on Hera than Hercules, etc.

Reply:
Isn't that fixing a problem that probably doesn't exist? I don't know what the optimum thread number is, but it shouldn't be hpc dependent. The one possible issue is the memory required to run more threads, but that hasn't been an issue so far on the hpc we use.

Reply:
it's probably not an issue for ctest, but it's absolutely an issue and machine dependent when running `make`. But, this was just a suggestion, not a requirement

Reply:
Scratch the above, I'm again selecting snippets of text and extrapolating the rest without reading .. I'm realizing this is for the test, not make.
@CoryMartin-NOAA , I don't see why make -jN should be machine dependent.

Reply:
@guillaumevernieres well Hera, for example, has fewer cores and less memory per login node than Hercules...

Reply:
> @guillaumevernieres well Hera, for example, has fewer cores and less memory per login node than Hercules...

I haven't played with this in a while, but the optimum N is well below the number of available threads we have access to, regardless of what hpc. Memory wise, 12 threads is an issue on my laptop but has never been an problem on hpc. One possible issue is how io may differ between hpc ... But I still think this is over-engineering.  KIS if we can!